src/preact/context.js

```diff
@@ -90,27 +90,9 @@ export function useAmpContext() {
  * Whether the calling component should currently be in the loaded state.
  *
  * @param {!Loading|string} loadingProp
- * @param {boolean} unloadOnPause
  * @return {boolean}
  */
-export function useLoading(loadingProp, unloadOnPause = false) {
```

Comment:
So the fix here is to have the callers take care of `unloadOnPause` themselves? We don't override the loading prop anymore.

Reply:
Correct. That lifecycle is simplified. In many cases the pausing action could be simpler, as I've done with Instagram.